pontoon/sync/formats/ftl.py

```diff
@@ -73,7 +78,7 @@ def get_comment(obj):
         for obj in self.structure.body:
             if type(obj) == ast.Message:
                 key = obj.id.name
-                translation = serializer.serialize_message(obj)
+                translation = serializer.serialize_entry(obj)
```

Comment:
Argh, sorry. I didn't see that comment (it was already closed). That can also be considered a bug (in GitHub). ;) And, sorry @jotes - you'll have to fix this line for the 3rd time now!